### DISCUSSION TITLE
Unable to Bridge to Lan on a system with Docker

### DISCUSSION THREAD

**Discussion by capsel22** [cockpit-project/cockpit-machines#1046] [Q&A]
Hi,

I am running ubuntu server 22.04 with Cockpit. This system initially had only Docker but now I needed to run a couple of VMs too. This is where the cockpit-machines came very useful.

However, I am having an issue where I cannot Bridge to Lan to my main eno1 interface. I can NAT and Directly Attach, but not bridge.

I am a fair Ubuntu newbie and I tried googling solution but all I found was in a scenario where Docker would fail to run when installed on KVM system, not the other way around.

Any ideas how I can get Bridge to LAN working?

Attaching screenshot of my interfaces and the lack of eno1 in dropdown list
![image](https://user-images.githubusercontent.com/38831192/230078700-557f9a5c-d9c9-4ca7-8c5c-0c1ede9d26f3.png)
![image](https://user-images.githubusercontent.com/38831192/230078858-8f794613-5017-43f5-9556-f6a19bacc54b.png)

**Comment by KKoukiou** [comment]
What you see there is expected. You can only use 'Bridge to LAN' for bridge interfaces. You can create a new bridge to your eno1 interface and then use this for your VMs.
